prow/jobs/images/Dockerfile.auto-tag

```diff
@@ -0,0 +1,23 @@
+FROM debian:buster-slim AS base
```

Comment:
And maybe rename this to `Dockerfile.release-tag` or `Dockerfile.controller-release-tag` depending on what you renamed the script file to above.